tests/security_features_test.go

```diff
@@ -141,6 +142,30 @@ var _ = Describe("SecurityFeatures", func() {
 				}
 				Expect(*container.SecurityContext.Privileged).To(BeFalse())
 			})
+
+			It("[test_id:5297]Make sure qemu processes are MCS constrained", func() {
+
+				By("Starting a VirtualMachineInstance")
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStart(vmi)
+
+				pod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+
+				qemuProcess, err := tests.ExecuteCommandOnPod(
+					virtClient,
+					pod,
+					"compute",
+					[]string{"/usr/bin/bash", "-c", "ps -efZ | grep [q]emu-kvm | awk '{print $1}'"},
```

Comment:
Is the `q` in brackets to avoid grepping the `grep` process? Cool trick!
Nit: we should probably `Expect()` the output of this to be 1 line only. To make sure that's true we could also grep for something bigger (like `/usr/libexec/[q]emu-kvm`).

Reply:
Updated this line to `"ps -efZ | grep [/]usr/libexec/qemu-kvm | awk '{print $1}'`

Reply:
yes, `q` in brackets to avoid grepping the grep process.